matRad_rayTracing.m

```diff
@@ -128,5 +129,9 @@
 % only take voxel inside the patient
 for i = 1:ct.numOfCtScen
     radDepthV{i} = radDepthCube{i}(V);
+    if nargout > 1
```

Comment:
actually now I wonder why we even use this. Just call the output argument radDepthCube and you won't have to do any query to assign radDepthsMat here, since radDepthCube is generated anyways.

Reply:
Yes, I do not think it makes any sense do not just rename the output argument. Changed it with the commit.